third_party/maya/lib/usdMaya/userExportedAttributesUI.py

```diff
@@ -335,6 +334,14 @@ def GetJsonDict(self):
         return result
 
     @staticmethod
+    def GetUserAttributeWriters():
+        if ExportedAttribute.userAttributeWriters == None:
+            import types
+            r = cmds.usdListUserAttributeWriters()
```

Comment:
We're in control of `usdListUserAttributeWriters` command, can't we ensure that it always returns a list?